### ISSUE TITLE
[FEATURE] Please bring back views in Listings

### ISSUE THREAD

**Issue by benjymouse** [etsy/open-api#122]
**Is your feature request related to a problem? Please describe.**
Views is one of the most useful parts of the Listing response and used extensively by our app. As per your 3.0.0-alpha-2021-06-29 release notes it has been taken away.

**Describe the solution you'd like**
See `views` in the response when fetching Listing objects.

**Describe alternatives you've considered**
Use v2 API until it is eventually shut down. If you can no longer provide views, please provide visits.

**Additional context**
Views are vital in tracking progress and calculating conversion rates.

**Comment by kbasali** [comment]
@benjymouse right. I'm trying to understand you removed this one but this feature is really important for our app too.

**Comment by shubham0804** [comment]
I agree 👍

**Comment by trungpv1601** [comment]
Please bring the **views** back 🥺

**Comment by benjymouse** [comment]
@harjas @capndesign  hey guys, sorry to bug you but any update on this? Thx

**Comment by harjas** [comment]
Hey @benjymouse! Sorry for the delay in our reply. The rationale behind removing the `views` field was that the data was coming from an internally deprecated data source. 

We understand this data is important and would love to support the `views` field. Migrating the accurate listing views data — which is visible in the seller dashboard — to the Open API would be a large undertaking. So, for the immediate general release at least, the views data will be unavailable.

If the removal of views is a pressing need for other developers as well, we can escalate the issue internally and report back. We're sorry for the inconvenience.

**Comment by benjymouse** [comment]
Thanks for the extra context @harjas 

How about providing conversion rate for each listing instead? Is that any easier?

**Comment by mattkingshott** [comment]
@harjas surely this thread is evidence enough that it’s a pressing need?!

**Comment by harjas** [comment]
@benjymouse I don't think we have conversion rate available as an exposable metric. We'll try to prioritize the work for views and get it out to you ASAP. Our intention was not to git rid of views entirely but we wanted to avoid a situation where we exposed the _old_ views metric, made the transition to the _new_ views data and sellers suddenly see a huge spike or a dip overnight.

**Comment by benjymouse** [comment]
Thanks @harjas I appreciate that you'll look it, I cannot emphasize enough how important this is for us. Another alternative, and useful addition, would be visits.

**Comment by VasilVelikov** [comment]
One more upvote for this issue.
The views/visits are a critical piece of data for our application as well. Without them we would have to deprecate several important tools on our website. This is a blocking issue for us to start any kind of migration to API v3.

**Comment by anhladang** [comment]
Hey @harjas . Do you have any information about the re-release of this information?

**Comment by amrutsabale** [comment]
One more upvote for this issue.
The views/visits are a critical piece of data for our application as well. Without them we would have to deprecate several important tools on our website. This is a blocking issue for us to start any kind of migration to API v3.

**Comment by VasilVelikov** [comment]
@harjas Do you have any clarity on the timeline for this improvement? It has been 3 months already and an update would be highly appreciated. Thanks.

**Comment by kbasali** [comment]
@harjas @etsyachristensen Do you have any good news for us?

**Comment by dbeleznay** [comment]
hi all - one more vote for including views or visit's in the API - it's important for my app as well.

**Comment by codyjmcguffie** [comment]
Another upvote for this issue. The listing views are SO SO SO important for our app. Please add listings views to V3. 

Thank you.

**Comment by natalyar16** [comment]
One more upvote for this issue.
The views are a critical piece of data for our application as well. Without them we would have to deprecate several important tools on our website. This is a blocking issue for us to start any kind of migration to API v3.
[@harjas](https://github.com/harjas) Do you have any information about the re-release of listing API with views field included?

**Comment by codyschneiderx** [comment]
Another upvote. This is essential to our app. 

@harjas please share an update on this if you can 🙏

**Comment by abhijit-bora** [comment]
Another upvote for this issue. The views from Etsy API v2 are so much essential for us and it gives much more value to our application. That helps many Etsy users gain a better understanding of the listed product. 
 
@harjas Please provide us some details about the addition of views in v3

**Comment by etsyachristensen** [comment]
Unfortunately Harjas has moved on to a different team. We are still investigating how to bring this back. The old method was no longer usable as it didn't accurately represent the view count.

**Comment by codyschneiderx** [comment]
@etsyachristensen Any update on this? 

Critical for a product feature in our app and if there is no longer an endpoint will have to change the app dramatically. 

Thank you for your help.

**Comment by etsyachristensen** [comment]
We are currently investigating how to bring this back effectively. It may take some time. But we do want to bring it back.

**Comment by NjaalWiik** [comment]
Any updates on this issue?

**Comment by rboydev** [comment]
Hey guys, are there any updates?

**Comment by emergingdzns** [comment]
Looks like views are now back in the listing endpoints. Etsy can close this one.

**Comment by Esatyilmaz0** [comment]
> Looks like views are now back in the listing endpoints. Etsy can close this one.

Hi bro, I can't access views. I try it many listing and not showing views property.
